easycukes-scm/src/main/java/com/worldline/easycukes/scm/stepdefs/SCMStepdefs.java

```diff
@@ -45,6 +46,21 @@
 public class SCMStepdefs {
 
     protected static final EasyCukesConfiguration<CommonConfigurationBean> configuration = new EasyCukesConfiguration<>(CommonConfigurationBean.class);
+    
+	/*
+	 *  Used to convert base url into scm base url 
+	 */
+	private static final String SCM_BASE_URL = "/portal";

```

Comment:
I used this to avoid sonar lint issue "Define a constant instead of duplicating this literal".  but you are right, to make this EasyCukes as a generic framework will log messages directly.